masonry/src/doc/06_masonry_concepts.md

```diff
@@ -0,0 +1,91 @@
+# Concepts and definitions
+
+This section describes concepts mentioned by name elsewhere in the documentation and gives them a semi-formal definition for reference.
+
+## Widget status
+
+The notion of widget status is somewhat vague, but you can think of it as similar to [CSS pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes).
+
+Widget statuses are "things" managed by Masonry that affect how widgets are presented.
+Statuses include:
+
+- Being hovered.
+- Having pointer capture.
+- Having local text focus.
```

Comment:
Below, you describe 'active text focus' and 'inactive text focus'. What's 'local text focus', though?

Reply:
I think it's the same. Basically there's "the widget is focused, but the window isn't", and "the widget and the window are both focused". I need to take another crack at explaining this.